### PR TITLE
LPS-199989

### PR DESCRIPTION
**Pull request by danielCamara02**
Passed on CI in this run: https://github.com/danielCamara02/liferay-portal/pull/62

Ticket: https://liferay.atlassian.net/browse/LPS-199989

### CODE REVIEW

**Comment by danielCamara02** [comment]
@suilin Since the test size has increased considerably, I put the test steps under task blocks. Hope this is ok

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsWebContent.testcase [hunk 1]
```diff
@@ -139,6 +139,162 @@ definition {
...
+
+			PortletEntry.changeLocale(locale = "de-DE");
+
+			WebContent.addWithStructureCP(webContentTitle = "WC Titel");
+
+			PortletEntry.publish();
+
+			WebContentNavigator.openWebContentTemplatesAdmin(siteURLKey = "Guest");
+
+			WebContentTemplates.addCP(templateName = "WC Template Name");
```
Comment (suilin) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsWebContent.testcase]: @danielCamara02, thanks for working on it, we'd better create the template with the structure that you created before, and the create order should be structure, template then web content, can you update it?

**Comment by suilin** [comment]
ci:forward

**Comment by suilin** [comment]
ci:stop

**Comment by suilin** [comment]
@danielCamara02, it looks good to me, seems there is a conflict, can you rebase and resend it? Thanks.

**Comment by danielCamara02** [comment]
ci:forward

**Pull request closed without merging**